### PR TITLE
[4.3] System menu - direct links

### PR DESCRIPTION
**Pull request by pe7er**
Pull Request to add **direct links** to the back-end admin menu "**System**".

### Summary of Changes

In Joomla 4 in the back-end the **System** menu link shows a Dashboard with all System related links.
This PR creates direct links under the System menu, while keeping its Dashboard view, similar like the other back-end menus.

### Testing Instructions

Click the **System** menu item in the back-end.

### Actual result BEFORE applying this Pull Request
The **System** menu item opens the **System Dashboard**

Screenshot from 2022-05-03 16-23-27

### Expected result AFTER applying this Pull Request

The **System** menu item now has an arrow and 4 tile icon:
Screenshot from 2022-05-03 16-39-30

The 4 tile icon behind the **System** menu item opens the **System Dashboard**
The **System** menu item itself opens an alphabetical list of menu headings from the System Dashboard items.
A click on each of the menu heading (of the System Dashboard items) will open the direct links.

Screenshot from 2022-05-03 16-21-44

Some of the sub menus like **Manage** and **Templates** use separators to easily distinguish the different similar items

Screenshot from 2022-05-03 16-22-06

### CODE REVIEW

**Comment by dgrammatiko** [comment]
The menu was specifically designed as a minimal entry point. You're reverting this and I'm guessing you have some UX/UI research to back it. Can you share it?

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 4a7c5e37ff79acd7c599170fd7562880f6ab44ad<br /><br />Tested successfully in Joomla 4.2.0-alpha3-dev of 3 May in Wampserver 3.2.8 using PHP 8.0.15<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by brianteeman** [comment]
why not as another option just as we already have an alternative ad,min preset which is basicaly a vertical version of the j3 menu

**Comment by dgrammatiko** [comment]
> why not as another option just as we already have an alternative ad,min preset which is basicaly a vertical version of the j3 menu


Also if the old school is really finding it difficult to adjust to j4 you can add a single input in the installation to set the profile people want: default to clean and minimal (current one), alternative, or this. My point is that some of us spent literally months on this, so a simple pr that reverts all that work is border line a goodbye message…

**Comment by pe7er** [comment]
@dgrammatiko 
> The menu was specifically designed as a minimal entry point. You're reverting this and I'm guessing you have some UX/UI research to back it. Can you share it?

It's a combination of eight months annoyance when installing Joomla 4 websites, combined with what I hear from other experienced Joomla back-end users.

To explain, lets describe the current "**System**" menu item. When you click on it you'll get a "**System Dashboard**". The "System Dashboard" has 8 "**Main Task Areas**" (Setup, Install, Templates, Maintenance, Manage, Information, Update, User Permissions). Those "Main Task Areas" have 31 different "**Tasks**" (e.g. Setup: Global Configuration, Install: Extensions, etc).

The problems with the current System menu and System Dashboard:
- The **System Dashboard** is **a bit cluttered** with information:
  - Too many "Main Task Areas" (8 Main Task Areas) too choose from
  - Some "Main Task Areas" have too many "Tasks" (e.g. Manage has 9 tasks).
  - You have to "scan" the whole page to find the right Task out of 31 options (yes, I know we need them all)
- The System Dashboard is **difficult to navigate**:
  - Navigating the items is difficult because they are on the whole page
  - Large mouse movements are needed to choose any of the Tasks on the right side of the screen
  - When you click the wrong Task, you need to move the mouse all over to the left, click "System", and move to the right to select another Task on the System Dashboard. And repeat 

My PR fixes that:
- The System Dashboard is still a bit cluttered but if offers **an alternative navigation**:
  - A click on "System" shows only the 8 "Main Task Areas"
  - A click on a "Main Task Area" opens a sub menu with all Tasks under that Main Task Area
  - Some tasks (e.g. under Manage, Templates) have separators that groups the Tasks in similar tasks
- The new System Menu is **easy to navigate**:
  - Navigating the items is easy because the Main Task Areas only show the Tasks when clicked
  - You do not need large mouse movements to choose any of the Tasks as you stay on the left side of the screen
  - When you click the wrong Task, you can easily click the next Task. The Tasks on the same Main Task Area are even still open and easy to click.

and a bonus:
After my PR the "**System**" menu item **is consistent with the rest of the menus** (Content, Menus, Components, Users): They all **work with the same navigation** (click opens sub items), and all have the **same Dashboard icon** with Dashboard functionality.

**Please test my PR** using the Patch Tester. Click on the "System", it's "Main Task Areas" and its underlying "Tasks". **And experience** that it improves the user experience because you can **faster select** the right Task, and **easily select another** Task from the same "Main Task Area", all with **less mouse movements**.

**Comment by brianteeman** [comment]
As I understand it the logic behind not having the menu cluttered with a hundred extra links was that all of the settings on the system page are site configuration and setup and will not be accessed on a regular basis once the site has been launched.

**Comment by dgrammatiko** [comment]
> what I hear from other experienced Joomla back-end users

The menu was simplified in order to be less intimidating for NEW users. The so called experienced backend users could switch to the alternative preset OR if they are really experienced they should know that they can create their own preset.

Your PR is reverting decisions that were made on the basis that Joomla should expand instead of appealing only to the existing user base that is constantly shrinking.

**Comment by HLeithner** [comment]
I think the system menu is a pain. So this may would reduce the pain for at least one part of it.

**Comment by woluweb** [comment]
tbh, every single time that I go to the System page, I (have to) use CTRL+F to find what I am looking for! :D
Example: I search for the word Extension, then I see that it is the Install Section, in the Manage Section and in the Update Section... and I can click on it.

So actually I had thought that a Filter possibility woudl make this System page less cumbersome (a Filter like when you Create a new Module: there you can type a keyword and filter on "copyright" to quickly find the module... which is called "footer" for example).

But I am also in favor of this PR.
It was indeed a good idea to have all those 31 Tasks (which are used most of the time when *building* a website) gathered under a unique System menu item, being the last one (instead of having System as the first menu item like in J3 etc etc :D )

Having an arrow next to "System" does not make it more complicated for a new Joomla user (all the other menu items have the same arrow...).
But it makes the daily work more efficient and more pleasant.

My 2 cents anyways :)

**Comment by HLeithner** [comment]
@dgrammatiko and @brianteeman I Mike your down votes but a solution would help us more... Because creating websites or maintaining them is a pain.

**Comment by brianteeman** [comment]
a solution already exists for those that think they need it. you can either replace the admin module with the alternate_preset or you can add the system_preset

**Comment by dgrammatiko** [comment]
@HLeithner I'm not against ADDING the expanded menu as ANOTHER preset but obviously I'm against making the expanded menu the default. Clean and simple is easier for newcomers and that is something that people adding more UI element should always be aware. Also ALL the links in the system are hardly need to be accessed in a daily basis similar to the control panel in Windows OS, the Settings in IOS, the System Preferences in MacOS, the settings in WP, etc, the pattern is not something invented here, it's the industry default!

**Comment by HLeithner** [comment]
> @HLeithner I'm not against ADDING the expanded menu as ANOTHER preset but obviously I'm against making the expanded menu the default. Clean and simple is easier for newcomers and that is something that people adding more UI element should always be aware. Also ALL the links in the system are hardly need to be accessed in a daily basis similar to the control panel in Windows OS, the Settings in IOS, the System Preferences in MacOS, the settings in WP, etc, the pattern is not something invented here, it's the industry default!

it doesn't change the fact that's a pain. You remembered the filter I proposed and you started to write? that would really help. Also the problem that you loose context when you any of the system menus makes it much harder. and tbh we have a menu on the last which is normally closed and only open when relevant.

Sure you don't need all of them and that's the biggest problem. normally when you create a website you may need 3-5 of them and it's always the same. looking at the industry standard, for example ribbons in ms office the relevant options you need are all on the "start" ribbon. We have a page with 50 buttions grouped in areas and not in usage (which is technically completely right but ux really hard especially for new comers. 

my bigger concern is that it's hardcoded into xml but maybe it's the only way I don't know.

**Comment by HLeithner** [comment]
> a solution already exists for those that think they need it. you can either replace the admin module with the alternate_preset or you can add the system_preset

Joomla should be easy by default and not after you load another preset (which is also not easy)

**Comment by dgrammatiko** [comment]
> Joomla should be easy by default and not after you load another preset (which is also not easy)

`Joomla should be easy` WE totally agree on that. The problem that all of you very happily forget is that you have 10 years of Joomla experience and you decide what is easy for your level as the default behaviour which obviously is not true for newcomers! Do we agree that 10 years experience and using Joomla to earn your salary is raising different concerns than a random fella that is trying Joomla for the first time? If so then the default should be the lean one as it's easier for them to navigate around and it's following the same pattern that all the systems (OS or other web platforms). Then the experienced and pros need further steps to align the menu to their liking, right? Well as I pointed earlier it's easily fixable:
- add a new preset, eg named expanded
- on the last page of the installation add a simple dropdown with `recommended`, `alternative`, `expanded`
- do the db change for the admin menu onChange
- Job done, everybody happy

**Comment by HLeithner** [comment]
@dgrammatiko when was the last time when more options has making something easier? Why do you think that it's easy to go to the system menu and get 50 items at once? including add module here button?

Of course I have 15+ years of joomla experience but that doesn't mean that I like it hard to do things. and that's also true for new comers. Because after the 3rd website they created they know exactly the 5 things what they need on the settings page but also have always to search for them, like peter or me and pretty sure many other experienced Joomla integrators.

I don't need the menu that's only for consistent (all other points of the menu works like this) and it's possible that decisions were wrong and have to be reconsidered. But this setting page is not easy to use at least I hope we are on the same page here.

**Comment by dgrammatiko** [comment]
> But this setting page is not easy to use at least I hope we are on the same page here.

So if the page is bad we won't fix the page but the menu. Sure, that's also how I like to solve problems...

**Comment by HLeithner** [comment]
> > But this setting page is not easy to use at least I hope we are on the same page here.
> 
> So if the page is bad we won't fix the page but the menu. Sure, that's also how I like to solve problems...

I don't care about the menu too much but it would make the menu consistent, and it seems nobody want to fix the settings page.

**Comment by brianteeman** [comment]
There are far bigger issues to worry about such as com_workflows being virtually unusable, com_schedules with nothing useful to use with it, mail templates that dont do what they say.

**Comment by dgrammatiko** [comment]
> I don't care about the menu too much but it would make the menu consistent

That's kinda problematic, the decision for the menu was done on the basis that Joomla should attract new users and reverting it return the CMS to the 2.5/3.0 era. As I said add another preset and give an option @ installation so pro users can land on the fully expanded menu without having to spend 20seconds to do the change in the module. I can patch the installation if that's the problem...

**Comment by pe7er** [comment]
> There are far bigger issues to worry about such as com_workflows being virtually unusable, com_schedules with nothing useful to use with it, mail templates that dont do what they say.

Sorry, but could you please keep the discussion about this PR. No need for whataboutism.
Thanks!

**Comment by brianteeman** [comment]
So sad to hear that

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 4a7c5e37ff79acd7c599170fd7562880f6ab44ad<br /><br />

>similar like the other back-end menus.

:+1:
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by akshitrattan** [comment]
I have tested this item :white_check_mark: successfully on 4a7c5e37ff79acd7c599170fd7562880f6ab44ad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by akshitrattan** [comment]
I have tested this item :white_check_mark: successfully on 4a7c5e37ff79acd7c599170fd7562880f6ab44ad<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by richard67** [comment]
The PR has 2 good tests so it could be set to RTC, but I don't really understand the code style changes introduced by this PR. Since when do we indent 2 tabs instead of one for each level of indentation in an XML file?

File: administrator/components/com_menus/presets/default.xml [hunk 1]
```diff
@@ -321,12 +321,265 @@
...
+		<menuitem
+				title="MOD_MENU_INFORMATION"
+				type="heading"
+		>
+			<menuitem
+					title="MOD_MENU_INFORMATION_WARNINGS"
+					type="component"
+					element="com_installer"
+					link="index.php?option=com_installer&amp;view=warnings
+"
```
Comment (sandewt) [inline administrator/components/com_menus/presets/default.xml]: Should be :

`link="index.php?option=com_installer&amp;view=warnings"`

**Comment by sandewt** [comment]
> Since when do we indent 2 tabs instead of one for each level of indentation in an XML file?

Well noticed, see also my previous comment. So my test is not successful.

**Comment by pe7er** [comment]
> but I don't really understand the code style changes introduced by this PR.

Sorry, I had not set the Joomla code style in my IDE.
I've fixed the code style issue. Thanks @richard67 @sandewt !

File: administrator/components/com_menus/presets/default.xml [hunk 2]
```diff
@@ -322,11 +322,260 @@
 
 	<menuitem
 		title="MOD_MENU_SYSTEM"
-		type="component"
-		element="com_cpanel"
+		type="heading"
+		icon="users"
```
Comment (sandewt) [inline administrator/components/com_menus/presets/default.xml]: Should be:
`icon="wrench"`

File: administrator/components/com_menus/presets/default.xml [hunk 3]
```diff
@@ -322,11 +322,260 @@
...
 		class="class:wrench"
-		link="index.php?option=com_cpanel&amp;view=cpanel&amp;dashboard=system"
-	/>
+		dashboard="system"
+	>
```
Comment (sandewt) [inline administrator/components/com_menus/presets/default.xml]: Should be: line 329
Two tabs

File: administrator/components/com_menus/presets/default.xml (hunk 1 above)
Comment (pe7er) [reply administrator/components/com_menus/presets/default.xml]: thanks!

File: administrator/components/com_menus/presets/default.xml (hunk 2 above)
Comment (pe7er) [reply administrator/components/com_menus/presets/default.xml]: Thanks!

File: administrator/components/com_menus/presets/default.xml (hunk 3 above)
Comment (pe7er) [reply administrator/components/com_menus/presets/default.xml]: Thanks!

**Comment by pe7er** [comment]
I've fixed the code style issue. Thanks @sandewt !

**Comment by sandewt** [comment]
I have tested this item :white_check_mark: successfully on 99f4eb6e83f2f351a6bec4e2f7079dd7c93830b2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by chmst** [comment]
my2cent: The system panel is a pain and needs improvements. Best would be individual settings as in quickicons panel, wherw users can hide or show blocs and can change colours or use icons and maybe can change ordering of submenus for better UX. 

But the proposed solution here in the default panel is not what is inteded for J4 backend.

**Comment by brianteeman** [comment]
> Best would be individual settings as in quickicons pane

There was a PR for that iirc

**Comment by Fedik** [comment]
@pe7er do you want menu to be comfortable to you or to your clients? (you cannot please both)
I think currently the menu more "end user" oriented, than developer. And that is a good thing.

As developer you can create one more menu module, with your favorite links.

I would not want changes in this PR to be merged, sorry.

**Comment by pe7er** [comment]
@Fedik Have you tested the PR to experience how it works and feels. And that it improves the usability?

Yes, when I login as **Super Admin** then I need a user friendly administrator menu by default. 
I don't want to click Search and having to search manually within 31 links on a page.

For my clients: when they are Manager, they don't even have the System menu.
And I use Joomla's core feature to create custom User Groups + Access Viewing Levels, in combination with Joomla's custom back-end feature to create a custom admin menu for them with only the menu items they need.

**Comment by Fedik** [comment]
tbh, in past I tried make something similar as you made here, for my own use (with custom preset), 
but then I noticed that it rarely need and only a while site set up, later it just dead menu.

**Comment by brianteeman** [comment]
@Fedik can you email me please brian at teeman dot net - whatever email address I have for you must be wrong

**Comment by richard67** [comment]
Would it be a solution to add it to https://github.com/joomla/joomla-cms/blob/4.2-dev/administrator/components/com_menus/presets/alternate.xml instead of https://github.com/joomla/joomla-cms/blob/4.2-dev/administrator/components/com_menus/presets/default.xml ? The default would not be changed, but for @pe7er 's use case one could use the alternative preset?

**Comment by sandewt** [comment]
> Yes, when I login as Super Admin then I need a user friendly administrator menu by default.
I don't want to click Search and having to search manually within 31 links on a page.

The mobile version also provides a better user experience.

**Comment by obuisard** [comment]
At first, I thought about an 'Advanced Main Menu' preset that users could switch to. But shouldn't Super Users be able to have advanced settings available from the start (if one considers showing a more advanced menu an 'advanced setting'), no matter newbie or senior?

**Comment by brianteeman** [comment]
I dont consider having this extra menu an advanced setting. It completely goes against everything that the admin in j4 was designed to be. Keep it simple. Dont put everylittle thing on every page. If you dont use it regularly you dont need it on every page. When was the last time you added a redirect or created a new template style?

**Comment by obuisard** [comment]
> I dont consider having this extra menu an advanced setting. It completely goes against everything that the admin in j4 was designed to be. Keep it simple. Dont put everylittle thing on every page. If you dont use it regularly you dont need it on every page. When was the last time you added a redirect or created a new template style?

In that way, why do we keep Mass Mail in the Users menu? 
We can go through the main menu and hand pick several items that users may not use regularly, yet they are present.

**Comment by brianteeman** [comment]
Why would anyone waste their time doing that when someone will come along later and piss all over their work

**Comment by pe7er** [comment]
The fact that someone has invested a lot of time in something that is later replaced by something better is annoying for the person who invested the time. But that should not be a reason not to improve something that benefits many users.

If the time invested in changes were a reason not to make improvements, we would still be on Joomla 1.0.x :smiley:

**Comment by obuisard** [comment]
I don't think that is what Peter @pe7er meant by writing this PR. There is no dismissal of previous work. But after over a year of heavy use, there may be areas of the interface users fill may be still improved. Nothing is set in stone and anyone should be able to give their opinions and propose solutions on areas one may think could use some tweaking.

**Comment by brianteeman** [comment]
This isnt a tweak - its a complete revert of a feature.

**Comment by obuisard** [comment]
The proposal does not remove the way system items are ordered and classified.
The proposal keeps the dashboard intact.
The proposal adds another possible way to access the items of the system dashboard.
The proposal adds consistency between all menu items. Now all of them have a consistent menu item and sub-menus + a direct access to the dashboard.

@pe7er @brianteeman I may have overlooked something, but so far this is how I see it.

**Comment by brianteeman** [comment]
Yes you missed the comments from Dmitri's. This was a deliberate decision by those that created the template.

**Comment by obuisard** [comment]
> Yes you missed the comments from Dmitri's. This was a deliberate decision by those that created the template.

And that is why I am on the fence here. There are good points on both sides of the argument.

**Comment by brianteeman** [comment]
BTW the cms release team say you have decided on the features for 4.3. what are they?

**Comment by brianteeman** [comment]
The noise from people clammering for this to merged or opening issues aboout the system menu is really deafening.

**Comment by obuisard** [comment]
> BTW the cms release team say you have decided on the features for 4.3. what are they?

This week we are reaching out to developers to get a state of their proposed solutions and to project mentors at Google Summer of Code. This will help us make a definitive list, which Shirielle and I will share real soon. First, we will announce them internally. No big secrets but we want to make sure 4.3 improves usability. That is why discussing on such PRs is important to us, so we keep in line with our goal.

**Comment by crystalenka** [comment]
I think that the answer here is perhaps a compromise.

Not having any sub-items is cleaner, but annoying for things you need to access frequently.

Having ALL the sub items as dropdown theoretically removes clicks—or at least pages—but makes it hard to use or find things because there's just so much.

I think that the ideal solution is to only include the links that people access the most frequently.

In my experience, the average user will need just a few of these pages on a regular basis:

- Global config
- Cache
- Install extensions
- Update extensions
- Manage plugins
- Site template styles (though I'm not sure that's something that needs to be accessed regularly either)

Basically, we want to offer the same options as the existing quick icons—these are what people need the most! Having it in the sub menu lets you access it from anywhere in the admin, which is convenient. The rest of the menu items are typically one-offs, so fine to let people hunt on the system dashboard on occasion. 

Having a multi-level dropdown makes any menu less efficient because then it's ends up even more effort to find things than the way it exists now.

In the long run, it may help to reorganize the system dashboard menus because right now it's not really clear where to find some things. But that's a conversation for another day. :)

**Comment by brianteeman** [comment]
Do you really use install extensions on a regular basis or just in the first days os building a site?

**Comment by crystalenka** [comment]
Sometimes during updates, an extension must be installed from scratch. But that would be solved by adding a button to the install screen from the updates page, like I think a recent issue or PR referenced.

To be fair, the quick access needs between site building mode and ongoing maintenance are very different. But adding some kind of toggle somewhere wouldn't necessarily be discoverable. It's worth thinking on.

**Comment by dgrammatiko** [comment]
@crystalenka read my comments: add a new preset expose it in the installation process (last page) everybody happy 😃

**Comment by angieradtke** [comment]
Sometimes I  do not get along with the navigation - still searching.
The reason for this is that I have been used to the fold-out navigation for years.  
I personally find this patch good, because it supports **my l**earned behavior.
 But how does it affect newcomers?

I am already in the middle of my life and I have difficulties to understand the navigation with its few contrasts. 
I can't find things because the contrast between the levels is  too low. And everything is looking the same.
With the horizontal navigation it was easier for me because there was also a spatial distance, we don't have yet.
What I'm saying is, we're dealing with two different problems. The design of the navigation and its content.

**Comment by crystalenka** [comment]
Angie, that's a good point and something to consider when we are working on the usability tweaks for Atum in J5?

Regarding the menu items, is everyone amenable to the compromise of including quick access items only in the dropdown here for System, **even if we don't yet know exactly which items** that would include?

One step at a time. :)

**Comment by angieradtke** [comment]
I did not want to suggest a change here , but only make clear why some have so much difficulty with the menu.

**Comment by brianteeman** [comment]
> Regarding the menu items, is everyone amenable to the compromise of including quick access items only in the dropdown here for System, even if we don't yet know exactly which items that would include?

personally no because its too subjective. it will always be something that there will be people who say they use something everyday and they never use something. 

This is why I fought so hard for the ability to create your own admin menu. The only part of that I would have done differently would be that instead of it being an option, the admin menu would always be generated and managed by the menu manager.

Changing to that (which is what I always wanted) would allow everyone to be happy.

**Comment by obuisard** [comment]
Thinking out loud...and that maybe for another discussion...
What if, in the installation process, we offered slightly different experiences based on the level of knowledge of Joomla for the person who installs it? Something like: I am a rookie/I am experienced with Joomla. Just a simple click to select on or the other.  
I know, that is one more option to add to the installer...
With that one option, we add one message saying that all experienced features will be accessible through Joomla (maybe an option in the global configuration that would turn on those experienced features once the rookie feels like it)
Then, based on the answer, the interface looks slightly different: menu presets are different and maybe other aspects of the interface.

**Comment by chmst** [comment]
> What if, in the installation process, we offered to select which 'experience' the user has? Something like: I am a rookie/I am experienced with Joomla. Just a simple click to select one or the other.

I suggested that years ago during development of atum. The argument against was: users who choose the "light" variant never will detect the countless features of Joomla.

My Opinion:
These Options are not part of the daily work when a site is online and stable and should not be expanded. But they are daily work for developers and could be a developer preset.

**Comment by obuisard** [comment]
Thank you @chmst, I suspected this was discussed before. Thank you all for your input.

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 5.0-dev. No new features will be merged into Joomla! 4.3 series. Joomla! 4.4 series is a bridge release to make migration from Joomla! 4 to 5 as smooth as possible.

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 5.1-dev.

**Comment by Heinz-R** [comment]
I have tested this item :white_check_mark: successfully on a1c808b59adb3beb347c41a2a0a6be1409cd4a53<br /><br />works perfect<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by crimle** [comment]
I have tested this item :white_check_mark: successfully on a1c808b59adb3beb347c41a2a0a6be1409cd4a53<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by dorisdreher** [comment]
I have tested this item :white_check_mark: successfully on a1c808b59adb3beb347c41a2a0a6be1409cd4a53<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by richard67** [comment]
RTC as it has 2 successful tests. But I will also set the RMDQ (release managers decision queue) label as there was some discussion which seems to show pros and cons for this PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37732">issues.joomla.org/tracker/joomla-cms/37732</a>.</sub>

**Comment by bembelimen** [comment]
Hello @pe7er , thank you for your contribution.

Today in the maintainer meeting, we agreed there is a use case for such a menu, under certain conditions, but the idea of the system menu is to hide less used functions, so it's not cluttering the backend menu.

What we really would like to see is as a preset to select this contruction when creating another custom menu. Would you be willing to invest some time to have this in the core?
